src/delaunay_core/interpolation.rs

```diff
@@ -524,7 +531,16 @@ where
                 // out edge of the current natural neighbor.
                 //
                 // The natural_neighbor_polygon.svg refers to this variable as `c0`, `c1`, and `c2`.
-                let current = last_edge.face().as_inner().unwrap().circumcenter();
+                let vertices = last_edge
+                    .face()
+                    .as_inner()
+                    .unwrap()
+                    .positions()
+                    .map(|p| p.sub(position));
+
+                let current = math::circumcenter(vertices).0;
+                last_edge.face();
```

Comment:
just for sure, does this line have any effect?

Reply:
Whoops! Thanks, that was a leftover from some debugging. Removed.